client/src/components/HandleRequestButton/HandleRequestButton.tsx

```diff
@@ -0,0 +1,60 @@
+import { IconButton, Menu, MenuItem } from '@mui/material';
+import { MouseEvent, useState } from 'react';
+import SettingsIcon from '@mui/icons-material/Settings';
+import { Status } from '../../interface/Booking';
+import { useBookings } from '../../context/useBookingsContext';
+import { useAuth } from '../../context/useAuthContext';
+import { useSnackBar } from '../../context/useSnackbarContext';
+
+interface PropTypes {
+  bookingId: string;
+}
+
+const HandleRequestButton = ({ bookingId }: PropTypes) => {
+  const [anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
+  const { setBookingStatus } = useBookings();
+  const { profile } = useAuth();
+  const { updateSnackBarMessage } = useSnackBar();
+
+  if (!profile.isSitter) {
+    return <></>;
+  }
```

Comment:
We seem to have removed this field from the model, just wanted to bring that to attention.